### PR TITLE
Editing KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert

### PR DESCRIPTION
**Pull request by apinnick**
Editing runbook in preparation for publishing downstream.

### CODE REVIEW

**Comment by apinnick** [comment]
@sradco @nunnatsa 
This is a pilot PR, so that I can show the other writers how to update a runbook file. Please review. Thanks.

**Review by nunnatsa — APPROVED** [review]

File: docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
 # KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert
+<!-- Edited: apinnick@redhat.com, Aug 2022 --> 
 
 ## Meaning
-The `KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert` alert means that the HypeConverged cluster operator (HCO) is running, but the `HyperConverged` custom resource does not exist for more than an hour.
+This alert fires when the HyperConverged Operator (HCO) runs for more than an hour without a `HyperConverged` custom resource (CR).
```
Comment (nunnatsa) [inline docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md:5]: Please revert `HyperConverged Operator` to `HypeConverged cluster operator`

**Review by nunnatsa — CHANGES_REQUESTED** [review]
Thanks for the PR. It looks great. Just one comment inline.

File: docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md (hunk 1 above)
Comment (apinnick) [reply docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md:5]: Thanks. I checked the [repo](https://github.com/kubevirt/hyperconverged-cluster-operator). I will add that to the "Guidelines" and update this PR.

**Review by nunnatsa — APPROVED** [review]

**Pull request merged**